cylc/flow/data_store_mgr.py

```diff
@@ -1366,7 +1367,7 @@ def insert_job(self, name, point_string, status, job_conf):
             name=tproxy.name,
             cycle_point=tproxy.cycle_point,
             execution_time_limit=job_conf.get('execution_time_limit'),
-            platform=job_conf.get('platform')['name'],
+            platform=job_conf.get('platform name'),
```

Comment:
I don't think `platform name` is set in the job conf.

E.G. take a look at the simulation conf:

https://github.com/cylc/cylc-flow/blob/master/cylc/flow/task_job_mgr.py#L1346

So I'm guessing this is always evaluating as `None`?

The `job_conf` thinngy is now documented (recent addition) here:

https://cylc.github.io/cylc-doc/nightly_8.2/html/plugins/job-runners/index.html